fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,142 @@
+# import datetime
+import json
+
+# import yaml
+import time
+from typing import Any
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from datadog_checks.base.utils.db import QueryManager
+# from requests.exceptions import ConnectionError, HTTPError, InvalidURL, Timeout
+# from json import JSONDecodeError
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+# metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
```

Comment:
These metrics are still in the code? Should this be commented out?

Reply:
@yzhan289 - This should be fine, since for testing I am just going to test on accuracy, so the code will just go through the metrics_list which is currently accuracy and skip over the other metrics.